<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSDK测试</string>
  </property>
  <widget class="QPlainTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="serialTag">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>串口：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>切换节目编号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>370</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="playIndex">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选择节目图片</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sendCount">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>370</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="sendCountTag">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>次</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serial">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>清除选择</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
